spanner/CHANGELOG.md

```diff
@@ -4,6 +4,24 @@
 
 [1]: https://pypi.org/project/google-cloud-spanner/#history
 
+## 0.30.0
+
+### Implementation Changes
+
+- The underlying autogenerated client library was re-generated to pick up new features and resolve bugs. (#4695)
```

Comment:
80 char limit or meh? (I have tried to uphold this. There is also the "semantic line breaks" approach.)

Reply:
Done (commit will appear during rebase)